css/templates/gallery/page.css

```diff
@@ -291,6 +291,16 @@ body {
   .v-lg-hide { display: none }
 }
 
+.gallery-hide-mobile {
```

Comment:
can't you use sm-hide and/or xs-hide

Reply:
Oh yeah, I forgot those were part of `core.css`. Done!